walker-backend/src/controllers/users.ts

```diff
@@ -9,9 +9,11 @@ class UserController {
     if (unique) {
       return res.status(400).json(`Account with email ${body.email} already exists.`)
     }
-    const userObj = body.user;
+    const user = new User({person_name: body.person_name,
+      pet_name: body.pet_name,
+      email: body.email})
 
-    const user = new User(userObj)
+    user.schema.methods.setPassword(body.password)
```

Comment:
when I tested this in postman, this line wasn't actually setting the salt & hash / they weren't getting saved in the database either

worked properly when I changed the line to this:
`user.setPassword(body.password)`

Reply:
I think this is how the guide I was using was able to use methods as well but I have type check errors when I try to do this, unsure how to fix

Reply:
hmm looking at docs now I think I can fix it